scripts/generateCssFlow.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path')
+const fs = require('fs')
+
+const distDir = path.resolve(__dirname, '../dist')
+
+if (!fs.existsSync(distDir)) {
+  throw new Error('dist folder not available')
+}
+
+const files = fs.readdirSync(distDir)
+
+files.forEach(file => {
+  const parsed = path.parse(file)
+  if (parsed.name.endsWith('.css')) {
+    const fullPath = path.resolve(__dirname, '../dist', file)
+    console.log('stubbing css file', fullPath)
```

Comment:
I we are adding JS to `scripts` we should include it in the linting config. This should complain about console. You can add `eslint-disable no-console` or just use ``process.stdout.write(`stubbing css file ${fullPath}\n`)``